design_system/src/main/java/org/the_chance/honymart/ui/composables/HoneyFilledButton.kt

```diff
@@ -42,12 +43,14 @@ fun HoneyFilledButton(
 
             )
     ) {
-        Text(
-            text = label,
-            style = Typography.bodyMedium,
-            color = contentColor
-        )
-
+        Loading(state = isLoading)
+        if (!isLoading) {
```

Comment:
It would be better if we used `animatedVisibilty` instead of `if condition` to show the text.